ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -49,8 +49,12 @@ def rsqrt_(x, name=None):
 def sqrt_(x, name=None):
     return ivy.inplace_update(x, sqrt(x))
 
-
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
```

Comment:
Since I made pull request last week my file is not sync with the main branch. That why I added subtract_